formgroups/contactpersons.html

```diff
@@ -25,7 +25,7 @@
           <div class="input-group has-validation">
             <div class="form-floating">
               <input type="text" class="form-control" id="input-contactperson-firstname" pattern="^[a-zA-ZäöüÄÖÜß \-]*$"
-                name="cpFirstname[]" />
+                name="cpFirstname[]"  onchange="checkMandatoryFields()"/>
```

Comment:
Und schon hier würde sich der Event Handler lohnen, weil du den Event Handler auf mehrere Elemente anwenden könntest. Nur mal ein Beispiel und absolut ungetestet würde das dann in etwa so aussehen:
```
  $('input[name="cpLastname[]"], input[name="cpFirstname[]"], input[name="cpPosition[]"], input[name="cpEmail[]"], input[name="cpOnlineResource[]"], input[name="cpAffiliation[]"], input[name="usw[]"], input[name="newField2[]"]').on('change', function() {
    checkMandatoryFields();
  });
```